src/organizations/src/main/java/engineering/everest/lhotse/organizations/domain/OrganizationAggregate.java

```diff
@@ -51,8 +52,8 @@ public OrganizationAggregate(CreateRegisteredOrganizationCommand command) {
     @CommandHandler
     void handle(PromoteUserToOrganizationAdminCommand command) {
         validateOrganizationIsEnabled();
-        Validate.isTrue(!organizationAdminIds.contains(command.getPromotedUserId()),
-                "User %s is already an admin of organization %s", command.getPromotedUserId(), id);
+        isTrue(!organizationAdminIds.contains(command.getPromotedUserId()),
+                "USER_ALREADY_ORGANIZATION_ADMIN", command.getPromotedUserId(), id);
```

Comment:
It could be useful if all the in18MessageKey constants are imported from the i18n package?

Reply:
:+1: